librfn/libopencm3/console_cdcacm.c

```diff
@@ -305,6 +303,18 @@ static enum usbd_request_return_codes cdcacm_control_request(usbd_device *dev,
 		 * even though it's optional in the CDC spec, and we don't
 		 * advertise it in the ACM functional descriptor.
 		 */
+		char local_buf[10];
+		struct usb_cdc_notification *notif = (void *)local_buf;
+
+		/* We echo signals back to host as notification. */
+		notif->bmRequestType = 0xA1;
+		notif->bNotification = USB_CDC_NOTIFY_SERIAL_STATE;
+		notif->wValue = 0;
+		notif->wIndex = 0;
+		notif->wLength = 2;
+		local_buf[8] = req->wValue & 3;
+		local_buf[9] = 0;
+		// usbd_ep_write_packet(0x83, buf, 10);
```

Comment:
What does this code do? With the write packet commented out it appears to be a nop.